packages/ui-extensions-server-kit/package.json

```diff
@@ -35,6 +35,8 @@
     "jest": "^27.1.0",
     "jest-websocket-mock": "^2.2.1",
     "mock-socket": "^9.0.5",
+    "react": "16.14.0",
+    "react-dom": "16.14.0",
```

Comment:
Does it error if you remove these two?

Reply:
No, storybook still runs properly if those two are removed